src/plugins/home/public/services/feature_catalogue/feature_catalogue_registry.ts

```diff
@@ -63,17 +64,21 @@ export class FeatureCatalogueRegistry {
   }
 
   public start({ capabilities }: { capabilities: Capabilities }) {
-    return {
-      get: (): readonly FeatureCatalogueEntry[] =>
-        [...this.features.values()]
-          .filter(entry => capabilities.catalogue[entry.id] !== false)
-          .sort(compareByKey('title')),
-    };
+    this.capabilities = capabilities;
+  }
+
+  public get(): readonly FeatureCatalogueEntry[] {
```

Comment:
Why did `get` move out of the plugin's public contract to be an API method of the plugin. Don't we need it to be consumed by other plugins?

Reply:
It was exposed because the home app was still part of the legacy platform while the registry already moved. Now both are back together in a single plugin and there is no need anymore to expose that information.